doc/api/colors_api.rst

```diff
@@ -14,42 +14,307 @@
    :no-members:
    :no-inherited-members:
 
-Classes
--------
+Color Conversion tools
+----------------------
+
+.. autosummary::
+   :toctree: _as_gen/
+   :template: autosummary.rst
+
+   is_color_like
+   same_color
+
+   hsv_to_rgb
+   rgb_to_hsv
+
+   to_hex
+   to_rgb
+   to_rgba
+   to_rgba_array
+
+   get_named_colors_mapping
+
+
+.. _norms_and_colormaps:
+
+Normalization and Colormapping of Continuous Data
+-------------------------------------------------
+
+Some `~.artist.Artist` classes can map an array of input data to RGBA
+values, (ex `~.axes.Axes.scatter` or `~.axes.Axes.imshow`).  The
+machinery for this is implemented via the `~.cm.ScalarMappable` base
+class in `~.cm` and the `~.Normalize` and `~.Colormap` classes in
+`~.colors` (this module).
+
+At the core, colormapping is going from a scalar value to a RGB tuple (formally
+:math:`f(x) : ℝ^1 \rightarrow ℝ^3`).  To effectively communicate through the
+color we want pick a :ref:`colormap suited to the data
+<sphx_glr_tutorials_colors_colormaps.py>`.  For continuous data types [#f1]_ a
```

Comment:
The outline here is a little non-linear, and seems to take a while before it settles into an order.  
- mapping
- choosing colormap
- mapping
- norm
- norm
- colormapping 

I think this could all be three paragraphs, or maybe 5 if you want to be expansive:

1) overview
2) norm step
3) colormap step

If you plan to talk about the "Curve through  space"  I wonder if a diagram would make this much clearer.  Expressing this with math formalisms without diagrams is going to lose your audience.